src/mpris.rs

```diff
@@ -118,7 +118,16 @@ fn get_metadata(playable: Option<Playable>) -> Metadata {
     );
     hm.insert(
         "xesam:url".to_string(),
-        Variant(Box::new(playable.map(|t| t.uri()).unwrap_or_default())),
+        Variant(Box::new(
+            playable
+                .map(|t| {
+                    format!(
+                        "https://open.spotify.com/track/{}",
```

Comment:
This will not work for podcast episodes, as the path changes. You could grab the `share_url()` which should give you the correct URL for tracks/podcasts.

Reply:
Okay, I will look in to this. Should `mpris:trackid` also be changed since it also has `spotify:track:` statically typed or does it stay the same with podcasts?

Reply:
Yes, good catch!